.github/workflows/build_linux_packages.yml

```diff
@@ -80,34 +80,26 @@ jobs:
           cmake --build build --target therock-archives therock-dist
           kill %1
 
-      # - name: Build Tarballs
-      #   run: |
-      #     (cd build && cpack -G TXZ)
+      - name: Test Packaging
+        run: |
+          ctest --test-dir build --output-on-failure
 
       - name: Report
         if: ${{ !cancelled() }}
         run: |
-          ls -lh build
+          echo "Full SDK du:"
+          echo "------------"
+          du -h -d 1 build/dist/rocm
+          echo "Artifact Archives:"
+          echo "------------------"
+          ls -lh build/artifacts/*.tar.xz
+          echo "Artifacts:"
+          echo "----------"
+          du -h -d 1 build/artifacts
+          echo "CCache Stats:"
+          echo "-------------"
           ccache -s
```

Comment:
FYI if you use https://github.com/hendrikmuhs/ccache-action, that has a post-job hook that reports the stats, for example: https://github.com/nod-ai/TheRock/actions/runs/13189325428/job/36821279513#step:22:2

The explicit approach you have here is fine - easy to understand and debug as needed. I like using the action since it lets me also handles restoring and saving, so the workflow code is simpler.

Reply:
Will keep that in mind. This workflow is getting long in the tooth and will need an upgrade soon.